packages/core/ui-settings/core-ui-settings-browser-internal/src/settings_service.test.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import * as Rx from 'rxjs';
+
+import { httpServiceMock } from '@kbn/core-http-browser-mocks';
+import { injectedMetadataServiceMock } from '@kbn/core-injected-metadata-browser-mocks';
+import { SettingsService } from './settings_service';
+
+const httpSetup = httpServiceMock.createSetupContract();
+
+const defaultDeps = {
+  http: httpSetup,
+  injectedMetadata: injectedMetadataServiceMock.createSetupContract(),
+};
+
+describe('#stop', () => {
```

Comment:
NIT/Optional: add a test asserting that `setup()` instantiate the two client with the correct parameters